<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditUser</class>
 <widget class="QWidget" name="EditUser">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>478</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit/View Member Info</string>
  </property>
  <widget class="QDialogButtonBox" name="saveCancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>166</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="privateTele">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Private Tele?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getUser">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Get User</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>146</width>
     <height>216</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="userEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>First Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="firstName">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Last Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lastName"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Tele. </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="telephone">
      <property name="inputMask">
       <string>DDDDDDDDDD; </string>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>146</width>
     <height>104</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="password"/>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string> New User Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="userName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>229</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Unit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="unitNum"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Move in Date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="moveInDate">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="date">
             <date>
              <year>2010</year>
              <month>3</month>
              <day>4</day>
             </date>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Pre-Coop Address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pastAddress">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>252</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Amount Owing:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="owing"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>501</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Committee:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="committee"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Position:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="position"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QCheckBox" name="over21">
        <property name="text">
         <string>Over 21yrs old?</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="movedOut">
          <property name="text">
           <string>Member moved out?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteUser">
          <property name="text">
           <string>Delete Member</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>saveCancel</zorder>
  <zorder>privateTele</zorder>
  <zorder>getUser</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>over21</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
